webapp/components/form/inputChips.js

```diff
@@ -131,13 +127,19 @@ const InputChips = props => {
   )
 }
 
+// TODO When refactoring InputChips:
+// 1. items and itemsLookupFunction become one required prop `items` array or function (see dropdown)
+// 2. `itemKeyProp` and `itemKeyFunction` become one required prop `itemKey` string or function (see dropdown)
+// 3. `itemLabelProp` and `itemLabelFunction` become required one prop `itemLabel` string or function (see dropdown)
+// 4: `dropdownAutocompleteMinChars` must be renamed to autocompleteMinChars
```

Comment:
other PR?

Reply:
yeah there'a a TODO above. this pull request is about Dropdown not input chips